app/views/foreman_virt_who_configure/configs/show.html.erb

```diff
@@ -16,10 +17,15 @@
 
     <div class="row">
       <div class="col-md-8">
-        <pre class="terminal"><%= @config.virt_who_config_file %></pre>
+        <div class="pull-right">
+          <%= link_to _('Edit this configuration'), edit_foreman_virt_who_configure_config_path(@config), :class => 'btn btn-default' %>
+          <%= link_to _('Download the script'), deploy_script_foreman_virt_who_configure_config_path(@config), :class => 'btn btn-default' %>
+          <%= link_to _('Copy to clipboard'), '#', :class => 'btn btn-default', :onclick => 'virt_who_copy_configuration_to_clipboard($("pre.terminal").html())' %>
```

Comment:
Replacing `'#'` with `nil` will remove href and the button will stop scrolling the page to the top.